utils/plots.py

```diff
@@ -0,0 +1,152 @@
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from sklearn.metrics import r2_score
+from sklearn.feature_selection import r_regression
+from scipy.stats import spearmanr
+from scipy.stats import pearsonr
+
+
+def plot_multiple_cdfs_with_medians(num_of_top_preds, dataframes_for_top_preds, labels_for_top_preds, lfc_values_list, labels, title_sufix):
+    """
+    Plots the CDFs for multiple sets of log2 fold change (LFC) values with median lines from multiple dataframes.
+    
+    Parameters:
+    - num_of_top_preds: Number of top predictions for each miRNA
```

Comment:
Do not require this param. Give it a default value. And if somebody wants to change it, they can.